pkg/api/alerting_test.go

```diff
@@ -66,7 +66,7 @@ func TestAlertingApiEndpoint(t *testing.T) {
 			})
 		})
 
-		loggedInUserScenarioWithRole("When calling GET on", "GET", "/api/alerts?dashboardId=1", "/api/alerts", m.ROLE_EDITOR, func(sc *scenarioContext) {
+		loggedInUserScenarioWithRole("When calling GET on", getMethod, "/api/alerts?dashboardId=1", "/api/alerts", m.ROLE_EDITOR, func(sc *scenarioContext) {
```

Comment:
Please use `http.MethodGet` instead of a custom variable.

Reply:
Good catch! Thanks!